src/content/api/module-methods.md

```diff
@@ -167,7 +168,7 @@ W> Module ID is a number in webpack (in contrast to NodeJS where it is a string
 
 ### `require.cache`
 
-Multiple requires to the same module result in only one module execution and only one export. Therefore a cache in the runtime exists. Removing values from this cache cause new module execution and a new export.
+Multiple requires in the same module result in only one module execution and only one export. Therefore a cache in the runtime exists. Removing values from this cache cause new module execution and a new export.
```

Comment:
I think this actually ment: "multiple requires of a specific module", so please revert

Reply:
```suggestion
Multiple requires of the same module result in only one module execution and only one export. Therefore a cache in the runtime exists. Removing values from this cache cause new module execution and a new export.
```

Reply:
Or change to of as suggested

Reply:
@EugeneHlushko yeah, it looks better with `of`.